packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -161,6 +161,7 @@ export const messages = {
     'SFDX: Start Function exited unexpectedly with code %s',
   force_function_invoke_text: 'SFDX: Invoke Function',
   force_function_invoke_tooltip: 'Send Request',
+  force_function_debug_invoke_tooltip: 'Debug Send Request',
```

Comment:
@ekapner fyi the tooltip (codelens) for debugging

Reply:
This works for me. How about you, @danyusf?

Reply:
Apologies, but I might need more clarity on what this really does. Does this basically do an invoke (via the Send Request button) and then attach the debugger?

Reply:
Yup, but in the opposite order: First attach the debugger, then do a function invoke. You don't have to know anything about attaching debugger or debug configuration, we do that for you.